.github/workflows/generate_third_party_licenses_workflow.yml

```diff
@@ -0,0 +1,35 @@
+name: Generate THIRD-PARTY-LICENSES
```

Comment:
we are calling this workflow both on `push tags` as described in its own yml and `wheels` workflow. Do we need to call this twice?

Reply:
We are using artifacts from this call for building arm wheels. When we will move building arm wheels to github actions, we can remove this call.